doc/spec/estimation/dml.rst

```diff
@@ -696,9 +696,7 @@ the case where this matrix has low rank: all the products can be embedded in som
 space and the cross-price elasticities is a linear function of these low dimensional embeddings. This corresponds
 to well-studied latent factor models in pricing. Our framework can easily handle this by using 
 a nuclear norm regularized multi-task regression in the final stage. For instance the 
-lightning package implements such a class:
-
-.. testcode::
+lightning package implements such a class::
```

Comment:
Released versions of lightning are currently not compatible with sklearn 0.23 (though this problem has been fixed in the lightning GitHub repo - see https://github.com/scikit-learn-contrib/lightning/pull/142).  We can revert this small change if there is a new PyPI release.